conda_build/post.py

```diff
@@ -320,7 +320,7 @@ def mk_relative(m, f):
     if not is_obj(path):
         return
 
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or ('bsd' in sys.platform):
```

Comment:
Please stop flooding this issue. I don't think spending 5 comments on how to rephrase a single line is justified. Both versions are correct, the complexity difference between both is non existent, there really is nothing to discuss here.